features/supplier/supplier_signs_framework_agreement.feature

```diff
@@ -0,0 +1,23 @@
+@supplier @supplier_account @gideon_test
```

Comment:
`@gideon_test` left in here.... We should probably have a new tag along the lines of `@supplier-esignature`?

Reply:
fixed in b2f82da